backend_new/src/validation-pipes/listing-update-pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { ArgumentMetadata, ValidationPipe } from '@nestjs/common';
+import { ListingsStatusEnum } from '@prisma/client';
+import { ListingUpdate } from '../dtos/listings/listing-update.dto';
+import { ListingPublishedUpdate } from '../dtos/listings/listing-published-update.dto';
+
+export class ListingUpdateValidationPipe extends ValidationPipe {
```

Comment:
I know this is a carry-over from the old backend, but there is a lot of duplication between this file and the listing-create-pipe. I wonder if we could somehow consolidate them?

Reply:
Nice call out, I went ahead and consolidated them. From talking to a friend who also uses nestjs (Treavor that gave the approval on this pr) there is actually another way to use pipes, and I think long term I want to try the approach he talked to me about. 

I can spin up a spike to give it a shot on a much smaller controller though, I think this pr is big enough as it is lol